src/includes/Schema02.js

```diff
@@ -0,0 +1,73 @@
+import { useState, useEffect } from 'react';
+import { secondSchema } from '.././data/sortedTerms.js';
+
+function Schema02() {
+    const [userData, setUserData] = useState(null);
+  
+    useEffect(() => {
+        async function fetchData() {
+          try {
+            const data = await secondSchema();
+            setUserData(data);
+          } catch (error) {
+            console.error('Error fetching data:', error);
+          }
+        }
+        fetchData();
+      }, []);
+  
+    return (
+      <div>
+        {userData ? (
+          <div>
+            <h2>ID: {userData.id}</h2>
+            <h2>Instagram</h2>
```

Comment:
why is Instagram hard coded?

Reply:
These are old schema testing files. I deleted them.